docs/changelog.md

```diff
@@ -13,9 +13,13 @@ with nothing else on the line.
       `sematic settings` command, and server settings are specified in the new
       `~/.sematic/server.yaml` file and accessed through the `sematic server-settings`
       command.
-    * [bugfix] Fix a bug deserializing `Union` values. Note that once you are using this release,
-      if you try to "rerun from here" using a pipeline that has `Union` values in it that were
-      produced prior to this version of Sematic, the rerun will likely fail with a serialization issue.
+    * [bugfix] BREAKING CHANGE: The examples' `README` files have been renamed to
```

Comment:
In case end-users already created examples using the template, they will get a Bazel error that the "README.md" file was not found.
The number of users should be low, and this changelog note is a low-effort way of providing some information about what to do